doc/libpmemobj/pmemobj_ctl_get.3.md

```diff
@@ -360,6 +360,11 @@ Extends the heap by the given size. Must be larger than *PMEMOBJ_MIN_PART*.
 
 This function returns 0 if successful, -1 otherwise.
 
+debug.heap.alloc_pattern | rw | - | int | int | - | -
+
+Sets or reads byte pattern for filling newly allocated memory. Disabled by
```

Comment:
Mention that `-1` is a special disable value.

Reply:
done

Reply:
Done